provisioner.go

```diff
@@ -357,21 +358,21 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 		},
 	}
 
-	pod, err := p.kubeClient.CoreV1().Pods(p.namespace).Create(helperPod)
-	if err != nil {
+	_, err = p.kubeClient.CoreV1().Pods(p.namespace).Create(helperPod)
+	if err != nil && !k8serror.IsAlreadyExists(err) {
```

Comment:
Nice solution!

Better add some comments here
```
# If it already exists due to some previous errors, the pod will be cleaned up later automatically
# https://github.com/rancher/local-path-provisioner/issues/27
```

Reply:
OK! Thank you for review.

Reply:
No problem. Next time you can add the code addressing the comment in the same commit since it's more clear to read in that way.

Merged, thanks.